.github/workflows/docs.yml

```
@@ -22,6 +22,7 @@ concurrency:
 
 env:
   USE_CACHE: ${{ (github.event_name == 'workflow_dispatch' && github.event.inputs.cache == 'true') || (github.event_name == 'pull_request') || (github.event_name == 'push') }}
+  DOCUMENTATION_CNAME: 'docs.pyvista.com'
```

This domain should be:

```suggestion
  DOCUMENTATION_CNAME: 'docs.pyvista.org'
```